api/db/changelog/db.changelog-183-new-user-recent-resource-table.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="neha" id="changelog-183-new-user-recent-resource-table">
+    <createTable tableName="user_recent_resources">
+      <column name="id" type="bigint" autoIncrement="true">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="user_id" type="bigint">
```

Comment:
user and workspace should have foreign key constraints

Reply:
oh shoot yes!

Reply:
Question: Since the workspace is no longer deleted in workbench, but only in firecloud, the cascade delete is not triggered for user_recent_resource. Do you agree, we should explicitly deleted all the resource in user_Recent_resources belonging to the workspace in case of delete since we have a limit of only 10 entries in the table.   (which means for every new entry the oldest is deleted which could be of active workspace)

Reply:
NVM thats happening in workspaceserviceimplL335

Reply:
I think that makes sense, but as part of a different PR / different story.  (maybe https://precisionmedicineinitiative.atlassian.net/browse/RW-6159 ?)

We want this PR to be as small as possible, and match current behavior exactly.